helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -335,11 +336,17 @@ internal fun OrtResult.getUnresolvedRuleViolations(): List<RuleViolation> {
     }
 }
 
+/**
+ * Return a copy with the [LicenseFindingCuration]s replaced by the given scope excludes.
+ */
+internal fun RepositoryConfiguration.replaceLicenseFindingCurations(curations: List<LicenseFindingCuration>):
+    RepositoryConfiguration = copy(curations = (this.curations ?: Curations()).copy(licenseFindings = curations))
```

Comment:
Better stick to the coding conventions: https://kotlinlang.org/docs/reference/coding-conventions.html#function-formatting

```kotlin
internal fun RepositoryConfiguration.replaceLicenseFindingCurations(
    curations: List<LicenseFindingCuration>
): RepositoryConfiguration = copy(curations = (this.curations ?: Curations()).copy(licenseFindings = curations))
```

Reply:
Done.